### PR TITLE
upgrade appchain.js and fix some bug

### PR DESCRIPTION
**Pull request by GyonGyon**

### CODE REVIEW

File: dist/migrate/index.js [hunk 1]
```diff
@@ -48,18 +47,19 @@ Migration.prototype.run = function(options, callback) {
...
-        var Migrations = resolver.require('./Migrations.sol')
-
-        if (Migrations && Migrations.isDeployed()) {
-          logger.log('Saving successful migration to network...')
-          return Migrations.deployed().then(function(migrations) {
-            return migrations.setCompleted(self.number)
-          })
-        }
-      })
+      // .then(function() {
```
Comment (ashchan) [inline dist/migrate/index.js]: Comment block of code should be deleted if they're not used.

File: dist/lib/testing/deployed.js [hunk 2]
```diff
@@ -31,9 +31,9 @@ var Deployed = {
 
   // Pulled from ethereumjs-util, but I don't want all its dependencies at the moment.
   toChecksumAddress: function (address) {
-    var web3 = new Web3();
+    var appchain = new AppChain();
```
Comment (ashchan) [inline dist/lib/testing/deployed.js]: See a lot of `var`. Possible to use `const` or `let` instead?

**Comment by ashchan** [comment]
Tests didn't pass.

File: dist/lib/testing/deployed.js (hunk 2 above)
Comment (GyonGyon) [reply dist/lib/testing/deployed.js]: Fixed

**Pull request merged**